ui-tests/src/test/java/org/openmrs/reference/FormTest.java

```diff
@@ -21,81 +21,83 @@
-			manageHtmlFormsPage.clickOnNewHtmlForm();
-			htmlFormsPage.createNewFormTest(name, description, version);
-		}
-		homePage.go();
-		homePage.goToManageForm();
-		if (!manageForm.addPresent()) {
+
+    private static String name = "newFormTest1";
+    private static String description = "description of new form";
+    private static String version = "1.2";
+    private HomePage homePage;
+    private AdministrationPage administrationPage;
+    private HeaderPage headerPage;
+    private ManageFormsPage manageForm;
+    private ManageHtmlFormsPage manageHtmlFormsPage;
+    private HtmlFormsPage htmlFormsPage;
+    private ClinicianFacingPatientDashboardPage patientDashboardPage;
+
+    @Before
+    public void setUp() throws Exception {
+        homePage = new HomePage(page);
+        assertPage(homePage.waitForPage());
+        headerPage = new HeaderPage(driver);
+        administrationPage = new AdministrationPage(page);
+        manageForm = new ManageFormsPage(driver);
+        htmlFormsPage = new HtmlFormsPage(page);
+        manageHtmlFormsPage = new ManageHtmlFormsPage(page);
+        patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);      
+    }
+
+    @Test
+    @Category(BuildTests.class)
+    public void initiateFormPresence() throws InterruptedException {
+        manageHtmlFormsPage = homePage.goToAdministration().clickOnManageHtmlForms();
+        if (manageHtmlFormsPage.getElementsIfExisting(By.xpath("//*[contains(text(), '" + name + "')]")).isEmpty()) {
+        	manageHtmlFormsPage.clickOnNewHtmlForm();
+        	htmlFormsPage.createNewFormTest(name, description, version);
+        }
+        homePage.go();
+        homePage.goToManageForm();
+        if (!manageForm.addPresent()) {
 			manageForm.delete();
 		}
-	}
-	
-	@Test
-	@Category(BuildTests.class)
-	public void addFormTest() throws Exception {	
-		initiateFormPresence();
-		manageForm.add();
-		manageForm.addLabel("Eye Report");
-		manageForm.addIcon("icon-align-justify");
-		manageForm.formIdFromUrl();
-		manageForm.save();
-		headerPage.clickOnHomeIcon();
-		assertNotNull("Eye Report", patientDashboardPage.FORM_EXIST);
-	}
-	
-	@Test
-	@Category(BuildTests.class)
-	public void editFormTest() throws Exception {
-		addFormTest();
-		homePage.goToManageForm();
-		manageForm.waitForPage();
-		manageForm.editPath();
-		manageForm.addLabel("Eye Test");
-		manageForm.save();
-		headerPage.clickOnHomeIcon();
-		assertNotNull("Eye Test", patientDashboardPage.FORM_EXIST);
-	}
-	
-	@Test
-	@Category(BuildTests.class)
-	public void deleteFormTest() throws Exception {
-		addFormTest();
-		homePage.goToManageForm();
-		manageForm.waitForPage();
-		manageForm.deletePath();
-		assertNotNull("Add", manageForm.ADD);
-	}
-	
-	@After
-	public void tearDown() throws Exception {
-	        homePage.go();
-	}
+    }
+
+    @Test
+    @Category(BuildTests.class)
```

Comment:
> Do we use tabs instead of space ?

Not sure whether your question was clear however, I followed the OpenMRS convention throughout the files as explained [above](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/530#discussion_r688242743) and showing practically how the file looks like after formatting it to the indention which I think is right one. I can as well share the suggested file lookup here [again](https://github.com/kdaud/openmrs-distro-referenceapplication/blob/RATEST-188/ui-tests/src/test/java/org/openmrs/reference/FormTest.java)